decoders/connector/milesight/ct101/v1.0.0/payload-config.jsonc

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "../../../../../schema/connector_details.json",
+  "description": "../description.md",
+  "install_text": "The Milesight CT101 LoRaWAN Sensor is a unique current monitoring solution designed for seamless integration into existing electrical systems. This self-powered sensor eliminates the need for batteries, offering long-term operation and reduced maintenance.\n\n**Key Features:**\n* **Self-powered:** Derives power directly from the measured circuit, eliminating battery dependence and ensuring continuous operation.\n* **Multiple current options:** Supports various current ranges to cater to diverse monitoring needs.\n* **High accuracy:** Delivers exceptional accuracy with a high sampling frequency of up to 3.3 kHz.\n* **Compact and easy installation:** Features a compact design and clamp-on functionality for quick and safe installation without interrupting the circuit.\n* **LoRaWAN connectivity:** Enables long-range, low-power communication for efficient data transmission.\n* **Threshold alarms:** Allows for setting thresholds to trigger alerts when current levels exceed or fall below predefined values.\n* **Remote monitoring**: Access real-time and historical data remotely through a LoRaWAN gateway and cloud platform.",
```

Comment:
When creating the install_text make sure the formatting is correct, talking in regards to the \n * etc...